src/shared/utils/createSearchParams.ts

```diff
@@ -0,0 +1,13 @@
+import { GetPaginatedMoviesParams } from '../../entities/movie/api/types';
+
+export const createSearchParams = (params: GetPaginatedMoviesParams): URLSearchParams => {
```

Comment:
use generic type for params
methods in shared folder shouldn't know about places or entities where they are used So they shouldn't import anything from features/entites/pages etc